src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/ProjectDifference.cs

```diff
@@ -13,6 +13,8 @@ internal enum ProjectDifference
         WorkspaceProjectAdded = 2,
         WorkspaceProjectRemoved = 4,
         WorkspaceProjectChanged = 8,
-        DocumentsChanged = 16,
+        DocumentAdded = 16,
+        DocumentRemoved = 32,
+        DocumentChanged = 64,
```

Comment:
We've got two enums here that basically do the same thing. I don't see it as a high priority to clean them up, but worth pointing out.